modules/ccs-gcp-customer-procedure.adoc

```diff
@@ -114,6 +114,24 @@ The following roles must be link:https://cloud.google.com/iam/docs/granting-role
 |Storage Admin
 |`roles/storage.admin`
 
+|Compute Load Balancer Admin
+|`roles/compute.loadBalancerAdmin`
+
+|Role Viewer
+|`roles/viewer`
+
+|Security Admin
```

Comment:
```suggestion
|IAM Security Admin
```
for consistency with OCP docs: https://docs.openshift.com/container-platform/4.15/installing/installing_gcp/installing-gcp-account.html